addons/frag/functions/fnc_dev_addRound.sqf

```diff
@@ -0,0 +1,93 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Lambda.Tiger
+ * This function initialize projectile tracking of a round so that it's path
+ * can be drawn in debug mode. It may optionally include hit / explode /
+ * deflected event handlers that spawn color coded spheres on each event,
+ * green / red / blue, respectively.
+ *
+ * Arguments:
+ * 0: Projectile to be tracked <OBJECT>
+ * 1: Add projectile hit/explode/defelceted event handlers <BOOL> (default: true)
+ * 2: Should the round track be blue. True results in blue traces, false in red <BOOL> (default: true)
+ *
+ * Return Value:
+ * Nothing Useful
+ *
+ * Example:
+ * [_projectile, false, false] call ace_frag_dev_addRound;
+ *
+ * Public: No
+ */
+
+params [
+    "_projectile",
+    ["_addProjectileEventHandlers", true],
+    ["_isTraceBlue", true]
+];
+
+if (_isTraceBlue) then {
+    GVAR(dev_trackLines) set [getObjectID _projectile, [[getPosATL _projectile], [0, 0, 1, 1]]];
+} else {
+    GVAR(dev_trackLines) set [getObjectID _projectile, [[getPosATL _projectile], [1, 0, 0, 1]]];
+};
+
+// event handler to track round and cleanup when round is "dead"
+[
+    {
+        if (isGamePaused) exitWith {};
```

Comment:
Should the code be the same as `FUNC(dev_trackObj)` or is there a specific reason why `accTime == 0` was omitted?
```suggestion
        if (isGamePaused || accTime == 0) exitWith {};
```

Reply:
It should be the same. I added it to dev_trackObj when we did the other changes earlier this week and forgot to change the other.